wrapper/CSharp/wolfCrypt-Test/wolfCrypt-Test.cs

```diff
@@ -670,6 +770,8 @@ public static void Main(string[] args)
             hash_test((uint)wolfcrypt.hashType.WC_HASH_TYPE_SHA512); /* SHA-512 HASH test */
             hash_test((uint)wolfcrypt.hashType.WC_HASH_TYPE_SHA3_256); /* SHA3_256 HASH test */
 
+            ecies_test(); /* ECIES test */
```

Comment:
Consider passing `const int keySize = 32;` as argument and running test for 48 (SECP381R1) and 66 (SECP521R1).